actions/contract-tests/README.md

```diff
@@ -27,7 +27,7 @@ jobs:
         run: $TEST_SERVICE_BINARY $TEST_SERVICE_PORT 2>&1 &
         
         # Here's the actual usage of this action!
-      - uses: launchdarkly/gh-actions/actions/contract-tests@contract-tests-v1.0.0
+      - uses: launchdarkly/gh-actions/actions/contract-tests@contract-tests-v1.1.0
```

Comment:
I think we could, but I was lazy.

Reply:
Oh, maybe so. I was thinking of updating the readme in a `workflow`, which is disallowed since it's in the `.github` folder. But since there are just in `actions`.. maybe it'd work fine.